src/site/stages/build/plugins/add-drupal-prefix.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable no-continue, no-param-reassign */
+
+const path = require('path');
+
+const {
+  ENABLED_ENVIRONMENTS,
+  PREFIXED_ENVIRONMENTS,
+} = require('../../../constants/drupals');
+
+const { logDrupal: log } = require('../drupal/utilities-drupal');
+
+function overwriteConflictingVagovContentFiles(files) {
+  for (const fileName of Object.keys(files)) {
+    const file = files[fileName];
+
+    if (!file.isDrupalPage) continue;
+
+    const pageFileDir = path.join(path.dirname(fileName));
+
+    const existingMarkdownIndexFile = path.join(pageFileDir, 'index.md');
+    const existingMarkdownFile = path.join(
+      pageFileDir,
+      '..',
+      `${path.basename(pageFileDir)}.md`,
+    );
+
+    const potentialConflicts = [
+      existingMarkdownIndexFile,
+      existingMarkdownFile,
+    ];
+
+    for (const vagovContentFile of potentialConflicts) {
+      if (files[vagovContentFile]) {
+        log(`Overriding conflicting vagov-content file: ${vagovContentFile}`);
+        // Maintain a reference, for the broken-link checker
+        file.overwrittenVagovContentPage = files[vagovContentFile];
+        delete files[vagovContentFile];
```

Comment:
If we don't delete Markdown files of the same name, they will overwrite the Drupal file of the same location. So, we delete them here and log a warning.